docs/source/conf.py

```diff
@@ -151,3 +159,52 @@
 html_css_files = [
     "css/custom.css",
 ]
+
+
+class TorchSynthParameterDoc(ClassDocumenter):
```

Comment:
The module parameter ranges were printing really gross -- this targets attributes that are lists of ModuleParameterRanges and iterates through and prints them nicely in a list with the range / default hyperparams

Reply:
Wow